### PR TITLE
Communication Frontend UI

### PR DESCRIPTION
**Pull request by HoJunHao2000**
# Todo
- [x] chatbox frontend
- [x] video chat frontend

# Changes made
1. Able to input and send multi-line text
(textbox can grow horizontally up till 5 lines)
2. On re-join will load back past messages
3. When users join will automatically connect to video.

# UI
Screenshot 2024-11-01 at 11 45 09 PM

### CODE REVIEW

File: communication-service/config.js [hunk 1]
```diff
@@ -6,5 +6,6 @@ module.exports = {
   SSL_CERT: process.env.COMM_SSL_CERT,
   USER_JWT_SECRET: process.env.USER_JWT_SECRET,
```
Comment (jeraldkiew) [inline communication-service/config.js:7]: if JWT_SECRET is used we should remove USER_JWT_SECRET

File: communication-service/controllers/socketController.js
Comment (jeraldkiew) [inline communication-service/controllers/socketController.js:1]: nice work on the message history 👍🏻

File: communication-service/config.js
Comment (jeraldkiew) [inline communication-service/config.js:1]: if JWT_SECRET is used we should remove USER_JWT_SECRET

File: communication-service/controllers/socketController.js
Comment (jeraldkiew) [inline communication-service/controllers/socketController.js:1]: nice work on the message history 👍🏻

**Review by jasonqiu212 — APPROVED** [review]
lgtm

**Review by jeraldkiew — APPROVED** [review]

**Pull request merged**